Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployWindow.cs

```diff
@@ -414,6 +416,21 @@ private void UnityBuildGUI()
                     }
                 }
                 EditorGUILayout.Space();
+                EditorGUILayout.BeginHorizontal();
```

Comment:
Use the scope types for inspectors

using (new EditorGUILayout.HorizontalScope())
{
}